src/fc/FlightPlan.cpp

```diff
@@ -70,6 +72,12 @@ void FlightPlan::update(AltFilter& filter){
 
                     sys.pyro.fire(e.squib);
 
+                    if (e.squib == Pyro::SquibA) {
```

Comment:
It's fine for now, but could be more generic. Some systems have more than 2 pyros.